src/pipelinedefinition.cpp

```diff
@@ -1065,14 +1065,17 @@ std::shared_ptr<TensorInfo> createOutputTensorInfoForPipeline(const std::string&
 }
 
 Status updateInputsInfoWithNodeConnection(tensor_map_t& inputsInfo, const TensorInfo& tensorInfo, const std::string& alias) {
-    auto newTensorInfo = std::make_shared<TensorInfo>(tensorInfo);
+    auto newTensorInfo = std::make_shared<TensorInfo>(alias, tensorInfo.getPrecision(), tensorInfo.getShape(), tensorInfo.getLayout());
     auto it = inputsInfo.find(alias);
     if (it != inputsInfo.end()) {
         if (!it->second->isTensorSpecEqual(*newTensorInfo)) {
             auto intersectionTensorInfo = it->second->createIntersection(*newTensorInfo);
             if (intersectionTensorInfo == nullptr) {
                 Status status = StatusCode::PIPELINE_INPUTS_AMBIGUOUS_METADATA;
-                SPDLOG_LOGGER_ERROR(modelmanager_logger, "Error validating pipeline: {}", status.string());
+                SPDLOG_LOGGER_ERROR(modelmanager_logger, "Error validating pipeline: {}; {}; {}",
```

Comment:
More explanation within error message

Reply:
The error is actually explained in depth, because we print `status.string()`.

Reply:
TODO: Add new line between tensors

Reply:
Added